docker-hub/domain-audit.md

```diff
@@ -29,6 +37,15 @@ To audit your domains:
   - Name: The name of the user.
   - Username: The Docker ID of the user.
   - Email: The email address of the user.
-  - Date Joined: The date the user created their Docker account.
 
-You can invite all the uncaptured users to your organization using the exported CSV file. For more details, see [Invite members via CSV file](../docker-hub/members.md/#invite-members-via-csv-file).
+You can invite all the uncaptured users to your organization using the exported CSV file. For more details, see [Invite members via CSV file](../docker-hub/members.md/#invite-members-via-csv-file). Optionally, enforce single sign-on or enable SCIM to automatically add users to your organization. For more details, see [Single Sign-on](../single-sign-on/index.md) or [SCIM](../docker-hub/scim.md).
+
+> **Note**
+>
+> Domain audit may identify accounts of users who are no longer a part of your organization. If you don't want to add a user to your organization and you don't want the user to appear in future domain audits, the account must be deactivated or the associated email address must be updated.
```

Comment:
If you don't want to add a user to your organization and you don't want the user to appear in future domain audits, you must deactivate the account or update the associated email address.